packages/e2e/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "root": "packages/e2e",
+  "sourceRoot": "packages/e2e/src",
```

Comment:
Each payment method can place its e2e tests here as a folder. Test framework automatically generates HAR files in this package as well.